packages/format-library/src/text-color/inline.js

```diff
@@ -65,7 +68,7 @@ const ColorPopoverAtLink = ( { isActive, addingColor, value, ...props } ) => {
 		if ( closest ) {
 			return closest.getBoundingClientRect();
 		}
-	}, [ isActive, addingColor, value.start, value.end ] );
+	}, [] );
```

Comment:
Which of these is changing to cause the cache to be bust and recompute the anchor?

Reply:
Kinda seems like the anchor calculation should be testing whether it's within the popover, and aborting if so.

Reply:
Seems like these values shouldn't be changing in the first place, which is something we should be addressing higher up than in this component itself. Notably, I observe that `value.start` and `value.end` each change to `undefined` at some points, which I don't think would ever be expected while this component is mounted.